frontend/appflowy_flutter/lib/window/app_window.dart

```diff
@@ -21,12 +25,21 @@ class AppWindow {
     }
 
     await windowManager.ensureInitialized();
+    final prefs = await SharedPreferences.getInstance();
+    final windowSize = json.decode(
+      prefs.getString('windowSize') ?? '{"height": 600.0, "width": 800.0}',
+    );
 
-    WindowOptions windowOptions = const WindowOptions(
-      minimumSize: Size(600, 400),
+    WindowOptions windowOptions = WindowOptions(
+      size: Size(windowSize['width'], windowSize['height']),
+      minimumSize: const Size(600, 400),
```

Comment:
Consider using the same constants here
```
static const double minWindowHeight = 400.0;
static const double minWindowWidth = 600.0;
```
Define them somewhere both classes have access to it.